packages/api-openfin/notification.html

```diff
@@ -0,0 +1,11 @@
+<h5 id="message">Message</h5>
```

Comment:
OpenFin notifications are html pages in their own right, and this is just injected inside the body of that page. I can still change it to be like this (but without the `<!DOCTYPE html>`) if you prefer?